### ISSUE TITLE
[JetSmartFilters] Filter Bricks Query Loop in Templates - PHP Class does not exist

### ISSUE THREAD

**Issue by hb81010** [CrocoBlock/suggestions#6652]
Jet SmartFilters: 3.1.0 
Bricks: 1.7

I noticed that filtering/sorting etc. from a Bricks Query Loop in a template may not work and the error message "PHP class does not exist" is given.

![grafik](https://user-images.githubusercontent.com/54399588/227045090-0cc519cb-a2ed-461e-959f-214f2e599093.png)

I could test and reproduce it here

- Product Archive Template
- Post Archive Template
- Single Product Template
- Single Post Template

I have copied the elements from the "Bricks Query Loop" and the filter 1:1 to a whole new page (no template) and it works perfectly there. Therefore, my guess would be that it generally occurs with templates.
But I haven't tested it yet with the Listing Grid instead of the Bricks Query Loop. Maybe the same problem is there.

**Comment by hb81010** [comment]
Update: The problem with filtering in the templates only happens with AJAX.

I have also tested the Listing Grid. Filtering in the template works there. But the featured image is replaced by a placeholder after filtering

![grafik](https://user-images.githubusercontent.com/54399588/227225562-87936cbb-88cb-4337-88e0-a308220c5073.png)

**Comment by tole011** [comment]
Any news on this?

**Comment by RagnarLodbrok94** [comment]
Hi, this issue has been resolved and will be the next release of JetSmartFilters.

**Comment by hb81010** [comment]
Thank you, filtering now works in the templates and the error message no longer appears.
I may have noticed a new bug now.

If I filter/sort etc. within a product category (template) with AJAX, the products from other categories are also displayed after filtering.

Before AJAX: (All 3 Products form a Category)

![grafik](https://user-images.githubusercontent.com/54399588/230965447-879664c7-da82-46db-a5af-ba5f139954c8.png)


After AJAX: (All Products)

![grafik](https://user-images.githubusercontent.com/54399588/230965650-570cfc54-6cfb-4d43-9ebc-9cd50351da21.png)


This is how my Bricks Loop Query looks. 

![grafik](https://user-images.githubusercontent.com/54399588/230965780-5d5b8089-0ecd-46ed-9769-f0c09fa44392.png)


I have only selected products. Apparently Bricks recognises which category you are in and then only outputs the products. I don't think there is anything with "Current Term" or something like that. 


As a check, I wanted to try it with "Page Reload" instead of AJAX. But then I am redirected to my startpage after confirming the button. On another page, which is not a template, I tested it and then both functions work.
